### ISSUE TITLE
Child links in navigation not accessible via keyboard

### ISSUE THREAD

**Issue by katknow** [omeka-s-themes/default#42]
Currently, the navigation fails the Accessibility Insights for Web assessment when child links are added to the navigation bar. While you can technically tab to them, it does so without opening the menu so it looks like you are tabbing to nothing. The navigation menu would need to be able to be opened and closed by keyboard to pass.

**Comment by kimisgold** [comment]
I've attempted to address this in b362181762db76bc6bca0bb4e2fffd52dc9761d5 and dc5c96bab8ff0ea6f2d39f4c9a7bcffbdbf56394 and the other previously referenced commits. The top nav should now be navigable by keyboard.

* The arrow buttons will open and close child menus when hovered in and out on as well as by click.
* On keyboard, tabbing out of a branch of the nav tree will close the child menu. Only one branch should be visible at a time.

This should be tested with screen readers on the major desktop and mobile browsers. I can cover android devices.

**Comment by katknow** [comment]
The child menu is not closing when tabbed through. Accessibility Insights advises that "focus should return to the original trigger component"

**Comment by kimisgold** [comment]
Could you take a screenshot or point me to the accessibility insights notice on this? I've looked at the examples in [W3C](https://www.w3.org/WAI/tutorials/menus/flyout/) and [Yale UI](https://yale-a11y.gitlab.io/ui-component-library/accessible-flyout-menu) that seem to demonstrate the same tabbing and toggling behavior here.

**Comment by kimisgold** [comment]
@aviannamiller Would you mind trying out this feature in Accessibility Insights and comparing to my references in the above comment? Just trying to get a clear read on the expected pattern for this kind of keyboard navigation.

**Comment by aviannamiller** [comment]
I am not experiencing the specific issue of the child menus not closing when tabbing through. Instead, VoiceOver is reading out child menus and allows one to navigate to the links, but tabbing does not **visually** "open" or "close" the child links (tested on Firefox, Chrome, and Safari). The issue I am seeing with all of this is a bit different and is well-summarized by [W3C's comments on Keyboard Users](https://www.w3.org/WAI/tutorials/menus/flyout/#keyboard-users): 

> Submenus should not open when using the tab key to navigate through the menu, as keyboard users would then have to step through all submenu items to get to the next top-level item.

VoiceOver is behaving in such a way that a user must tab through all child menu items, which are read out, in order to get to the next parent (not sure if the same is true when tested with NVDA or JAWS). To remedy this, it seems that using a button as a toggle (see [Use button as toggle](https://www.w3.org/WAI/tutorials/menus/flyout/#use-button-as-toggle)) would allow for screenreader users to navigate from parent to parent without the readout of each child item unless the button is activated.

Let me know if I am missing something pertinent to the original issue.

**Comment by kimisgold** [comment]
That's a good note. I think the w3c pattern here mainly applies to the flyout menu level here, which would mainly be the first two while all subsequent levels are displayed as nested lists with levels indicated by indentation. The megamenu pattern, hovering to reveal a complex navigation tree, was popular many years ago for providing quick overviews of available content. I think I lean towards it being overwhelming now, and as that excerpt says, a lot to navigate through via keyboard. I'll apply submenu toggles for those as well.

**Comment by aviannamiller** [comment]
In revisiting this, the toggles look great. That aspect seems to be settled at this point.

I am seeing one small styling issue in moving from an open mobile menu and widening the viewport. In this case, the menu items with toggles gain height, separating out the nav item's label:

<img width="644" height="146" alt="Detail of a site's menu. Four nav items listed in a menu bar, two of which have toggles and are in slightly taller containers, the item names breaking on to new lines" src="https://github.com/user-attachments/assets/78cb8c90-88a8-4a06-b857-1efe721b0c81" />
